tests/test_machine.py

```diff
@@ -89,3 +89,22 @@ def test_history_state():
     on_second_state = fan.transition(off_state, "POWER")
 
     assert on_second_state.value == "fanOn.second"
+
+
+
+final = Machine(
```

Comment:
Can you move this inside the test itself?

Reply:
Done